iree/test/e2e/regression/BUILD

```diff
@@ -48,6 +48,7 @@ iree_lit_test_suite(
             "dynamic_linalg_matmul_on_tensors_fuse_0.mlir",
             "dynamic_linalg_matmul_on_tensors_fuse_1.mlir",
             "dynamic_linalg_matmul_on_tensors_fuse_2.mlir",
+            "linalg_ext_ops.mlir",
```

Comment:
seems like this could go next to `linalg_ops.mlir` in `BACKEND_TESTS`?

```diff
BACKEND_TESTS = [
    "linalg_ops.mlir",
+   "linalg_ext_ops.mlir",
]
```

then no need to add explicitly to the exclude here or srcs below

Reply:
or remove `BACKEND_TESTS` as a list if it only has one element